### ISSUE TITLE
more than an experiment?

### ISSUE THREAD

**Issue by staugaard** [alloy/MotionData#3]
MotionData looks cool.

Will it ever be more than an experiment, or have you dropped it?

**Comment by alloy** [comment]
It has definitely not been dropped. If it will ever be more than an experiment has to be seen, though, I think that will mainly have to do with wether or not people start using it for real apps.

If you would like to do so, please let us know, but be prepared to roll up your own sleeves to work on the lib itself while you’re working on your app.

HTH :)

**Comment by 0x1b2c** [comment]
But if we want people start using it for real apps, we should at least make it a library rather than an experiment app, right?

**Comment by alloy** [comment]
@rainux True. What I meant is that I did not want to give potential users wrong expectations. However, if you/someone would provide a patch that makes it into a proper lib and gem, I will merge it :)

**Comment by n00shie** [comment]
Just wanted to chime in and say we have a team working on building a RubyMotion app, and we are looking into potentially using MotionData. So there is that much demand at least ;)

I will see if I can contribute something useful in the near future.

**Comment by alloy** [comment]
@n00shie I would love to see this being used in an actual app and taken in a more useful direction. The reason I have not made a gem out of it myself is because it feels like endorsing it in such a way where a user could somewhat expect it was battle-tested. Therefore, I’ll be looking forward to your contributions :)
